utils/remote.c

```diff
@@ -645,6 +647,22 @@ static void addDays(tr_variant* args, tr_quark const key, char const* arg)
     }
 }
 
+static void addLabels(tr_variant* args, const tr_quark key, const char* arg)
+{
+    tr_variant* labels = tr_variantDictAddList(args, key, 10);
```

Comment:
To allow for multiple `-L`/`--labels` arguments (just in case), check if the key already exists first.